core/templates/dev/head/pages/collection_editor/editor_tab/CollectionEditorTabDirective.js

```diff
@@ -20,7 +20,7 @@ oppia.directive('collectionMainTab', [function() {
   return {
     restrict: 'E',
     scope: {},
-    templateUrl: 'inline/collection_main_tab_directive',
+    templateUrl: 'collectionEditor/editorTab',
```

Comment:
This style wasn't discussed in the frontend refactor doc. We should discuss it (I'm fine with LGTMIng it here, but I want to understand the thoughts later on).